roles/rsyslog/roles/output_roles/files/defaults/main.yml

```diff
@@ -34,73 +34,50 @@ __rsyslog_conf_files_output_rules:
-          {% else %}
-          {{ element.facility }}.{{ element.severity }} {{ element.path }}
-          {% endif %}
-          {% else %}
-          {% if element.exclude is defined %}
-          *.{{ element.severity }};{{ element.exclude }} {{ element.path }}
-          {% else %}
-          *.{{ element.severity }} {{ element.path }}
-          {% endif %}
-          {% endif %}
-          {% else %}
-          {% if element.facility is defined %}
-          {% if element.exclude is defined %}
-          {{ element.facility }}.*;{{ element.exclude }} {{ element.path }}
-          {% else %}
-          {{ element.facility }}.* {{ element.path }}
-          {% endif %}
-          {% else %}
-          {% if element.exclude is defined %}
-          *.*;{{ element.exclude }} {{ element.path }}
-          {% else %}
-          *.* {{ element.path }}
-          {% endif %}
-          {% endif %}
-          {% endif %}
-          {% endif %}
-
+          {% for element in rsyslog_output_files %}
+          {%   if element.path is defined %}
+          {%     if element.exclude is defined %}
+          ruleset(name="{{ element.name }}") {
+            {{ element.facility | d('*') }}.{{ element.severity | d('*') }};{{ element.exclude }} {{ element.path }}
+          }
+          {%     else %}
+          ruleset(name="{{ element.name }}") {
+            {{ element.facility | d('*') }}.{{ element.severity | d('*') }} {{ element.path }}
+          }
+          {%     endif %}
+          {%   else %}
+          ruleset(name="{{ element.name }}") {
+              # Log all kernel messages to the console.
+              # Logging much else clutters up the screen.
+              kern.*                                                 /dev/console
+
+              # Log anything (except mail) of level info or higher.
+              # Don't log private authentication messages!
+              *.info;mail.none;authpriv.none;cron.none                {{ rsyslog_system_log_dir }}/messages
+
+              # The authpriv file has restricted access.
+              authpriv.*                                              {{ rsyslog_system_log_dir }}/secure
+
+              # Log all the mail messages in one place.
+              mail.*                                                  -{{ rsyslog_system_log_dir }}/maillog
+
+              # Log cron stuff
+              cron.*                                                  -{{ rsyslog_system_log_dir }}/cron
+
+              # Everybody gets emergency messages
+              *.emerg                                                  :omusrmsg:*
+
+              # Save news errors of level crit and higher in a special file.
+              uucp,news.crit                                          {{ rsyslog_system_log_dir }}/spooler
+
+              # Save boot messages also to boot.log
+              local7.*                                                {{ rsyslog_system_log_dir }}/boot.log
```

Comment:
I hope this huge inline of a config file will get changed in some subsequent PR

Reply:
> I hope this huge inline of a config file will get changed in some subsequent PR

Yes, this is going to be in a template file in the following pr.  (I.e.,  https://github.com/nhosoi/logging/pull/10, which will be submitted to this repo once this pr is merged.)